src/common-css/_common-mixins.scss

```diff
@@ -936,3 +936,100 @@ $sap-font-sizes: (
   }
   @content;
 }
+
+// FOCUS
+@mixin sap-fiori-focus($offset: -0.1875rem, $alternative: false) {
+  &:focus,
+  &.is-focus {
+    @if $offset != 0 {
+      outline-offset: $offset;
```

Comment:
If the dev passes 0 here, I would think we want to set the outline offset to 0? If they deliberately pass 0 and this `if` is skipped over, the outline-offset could be unintentionally set by some other selector

Reply:
We always had this mixin like this. It makes sense what you said so I removed the check